### ISSUE TITLE
Refactor resource list templates

### ISSUE THREAD

**Issue by holandes22** [holandes22/rafee#66]
Currently we use the resource-list component, but this component is not really well implemented, it deals with stuff that really needs to be done in the template level.

The idea of the component was to avoid code duplication in templates due to handlebars limitation to handle inheritance (for example, header and footers are always the same)

Once this is done, open a new ticket to deal with code dup, maybe using https://github.com/shannonmoeller/handlebars-layouts

**Comment by holandes22** [comment]
probably closes #55 and #49
